guides/common/attributes.adoc

```diff
@@ -130,7 +136,8 @@ ifdef::katello,foreman-el[]
 :ansible-namespace: `theforeman.foreman`
 :ansible-namespace-example: `theforeman.foreman._module_name_`
 :awx: AWX
-:BaseURL: https://docs.theforeman.org/nightly/
+:SiteURL: https://docs.theforeman.org
```

Comment:
Can you explain what this does that wasn't possible beforehand?

Reply:
it made it hard to refer to the previous version of the docs, like how to backup the previous version